### PR TITLE
Gatebox fix v2

### PR DESCRIPTION
**Pull request by jkarnasiewicz**
1. This solution don't require additional changes in homeassistant/components/blebox/const.py in comparison to version 1, but it's more obscure and ugly.
2. I left in code comments, additional notes and questions.
3. The main idea for this changes was to avoid or minimize modifications in Feature, and Cover class(or other common classes), and focused on changing GateBox class.

### CODE REVIEW

File: blebox_uniapi/cover.py [hunk 1]
```diff
@@ -2,11 +2,21 @@
 from .feature import Feature
 
 
-class Slider:
+class Gate:
+    """Previous class name: Slider, is it problem if I renamed it ?"""
```
Comment (bbx-jp) [inline blebox_uniapi/cover.py:6]: It's okay.

**Comment by gralin** [comment]
Should fix: https://github.com/blebox/blebox_uniapi/issues/122

**Review by bbx-jp — CHANGES_REQUESTED** [review]
Up

File: blebox_uniapi/cover.py [hunk 2]
```diff
@@ -91,12 +91,45 @@ def read_has_stop(self, alias, raw_value, product):
...
+
+        if current == 0:  # closed
+            return 3  # closed (lower/left limit)
+
+        if current == 100:  # opened
+            return 4  # open (upper/right limit)
+
+        # I don't know if I can leave it in that way,
+        # basically it will be returned while 'current == 50'
+        return 2  # manually stopped
```
[the remark this answers is not in the document]
Comment (jkarnasiewicz) [reply blebox_uniapi/cover.py]: Done.

**Pull request merged**